pygenie/adapter/genie_3.py

```diff
@@ -334,6 +335,19 @@ def get_info_for_rj(self, job_id, job=False, request=False,
 
             ret['client_host'] = execution_data.get('hostName')
 
+        if output or get_all:
+            output_data = self.get(job_id,
+                                      path='output',
+                                      if_not_found=dict(),
+                                      timeout=timeout)
+            output_files = output_data.get('files', [])
```

Comment:
May want to do `output_data.get('files') or list()` because if the `files` key is present but has a value of `None`, `None` will be returned.